packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -64,5 +64,6 @@
   "force_diff_against_org": "SFDX: Diff File Against Org",
   "force_analytics_template_create_text": "SFDX: Create Sample Analytics Template",
   "experimental_deploy_retrieve_description": "Enable performance enhancements to the deploy/retrieve experience.",
-  "experimental_apex_library_description": "Use Apex Library for Apex CLI commands"
-}
+  "experimental_apex_library_description": "Use Apex Library for Apex CLI commands",
+  "experimental_support_functions": "Enable building and managing Salesforce functions from VSCode"
```

Comment:
@ekapner

Reply:
`VSCode` should be `VS Code`

Reply:
Yep, just adding a space is my only feedback